### DISCUSSION TITLE
.csv import freezes after 49 rows

### DISCUSSION THREAD

**Discussion by shotleybuilder** [supabase/supabase#22601] [Questions]
Hi
My .csv. import of 30Mb and 1400 records gets to 49 records and then freezes (<2% complete).  Is there anything I might be able to do to resolve this?  My alternative is to write a script and upload via the REST api.  Clearly, .csv would save me a little bit of time.  Thanks
![Screenshot from 2024-04-09 15-48-49](https://github.com/supabase/supabase/assets/1798573/20f54105-55e8-4a47-87e7-87fa723cdfcf)

**Comment by GaryAustin1** [comment]
Any errors in the Postgres logs?

**Reply by shotleybuilder** [reply]
Hi, no, the only errors are ones I recognise re: invalid input syntax. These were fixed and enabled the import of the 49 records at least.  No problem - I'm exploring options.

**Reply by GaryAustin1** [reply]
You might just use PSQL and its copy command with database port.

**Reply by shotleybuilder** [reply]
Yeah, I did that too, but I've lots of columns and column ordering has to be specified in COPY to work - I might bite the bullet and list all those columns :)
